src/components/dashboard/container/DictionariesDisplay.jsx

```diff
@@ -4,117 +4,132 @@ import propTypes from 'prop-types';
+import AddDictionary from '../components/dictionary/AddDictionary';
 
 export class DictionaryDisplay extends Component {
-    static propTypes = {
-      fetchDictionaries: propTypes.func.isRequired,
-      dictionaries: propTypes.arrayOf(propTypes.shape({
-        dictionaryName: propTypes.string,
-      })).isRequired,
-      isFetching: propTypes.bool.isRequired,
-      searchDictionaries: propTypes.func.isRequired,
-      clearDictionaries: propTypes.func.isRequired,
-      fetchDictionary: propTypes.func.isRequired,
-      organizations: propTypes.arrayOf.isRequired,
+  static propTypes = {
+    fetchDictionaries: propTypes.func.isRequired,
+    dictionaries: propTypes.arrayOf(propTypes.shape({
+      dictionaryName: propTypes.string,
+    })).isRequired,
+    isFetching: propTypes.bool.isRequired,
+    searchDictionaries: propTypes.func.isRequired,
+    clearDictionaries: propTypes.func.isRequired,
+    fetchDictionary: propTypes.func.isRequired,
+    organizations: propTypes.arrayOf.isRequired,
+  };
+  constructor(props) {
+    super(props);
+    this.state = {
+      searchInput: '',
+      dictionaryViewOptions: 'user',
+      show: false,
     };
-    constructor(props) {
-      super(props);
-      this.state = {
-        searchInput: '',
-        dictionaryViewOptions: 'user',
-      };
-      autoBind(this);
-    }
+    autoBind(this);
+  }
+
+  componentDidMount() {
+    this.props.fetchDictionaries();
+  }
 
-    componentDidMount() {
-      this.props.fetchDictionaries();
+  onSubmit(event) {
+    event.preventDefault();
+    const { user } = this.state;
+    if (user) {
+      this.setState({ dictionaryViewOptions: 'user' });
+    } else {
+      this.setState({ dictionaryViewOptions: 'organizations' });
     }
+    this.props.clearDictionaries();
+    this.props.searchDictionaries(this.state.searchInput);
+  }
 
-    onSubmit(event) {
-      event.preventDefault();
-      const { user } = this.state;
-      if (user) {
-        this.setState({ dictionaryViewOptions: 'user' });
-      } else {
-        this.setState({ dictionaryViewOptions: 'organizations' });
-      }
+  onSearch(event) {
+    const { value, name, checked } = event.target;
+    const trueValue = event.target.type === 'checkbox' ? checked : value;
+    this.setState({ [name]: trueValue });
+    if (name === 'searchInput' && value.length <= 0) {
       this.props.clearDictionaries();
-      this.props.searchDictionaries(this.state.searchInput);
+      this.props.fetchDictionaries(value);
     }
+  }
 
-    onSearch(event) {
-      const { value, name, checked } = event.target;
-      const trueValue = event.target.type === 'checkbox' ? checked : value;
-      this.setState({ [name]: trueValue });
-      if (name === 'searchInput' && value.length <= 0) {
-        this.props.clearDictionaries();
-        this.props.fetchDictionaries(value);
-      }
-    }
+  handleHide = () => this.setState({ show: false });
+  handleShow = () => this.setState({ show: true });
```

Comment:
What are you trying to hide or show with these methods (`handleHide`, `handleShow`)?
Though it is understandable form the content of the methods that you are switching the value of a state variable(`show`). It would be better if the name of the methods is more explanatory as to what the methods are doing, this would ensure clarity of purpose when this methods are passed to other components as` props`.

Reply:
Same for these it will be another PR for refactoring code

Reply:
Understood.